src/patient/patient-search.service.ts

```diff
@@ -1,37 +0,0 @@
-import { Injectable } from '@angular/core';
-import { delay, of } from 'rxjs';
-
-@Injectable({
-  providedIn: 'root',
-})
-export class PatientSearchService {
-  constructor() { }
-  columns$ = of(['mrn', 'name', 'dob', 'location']);
```

Comment:
I wrapped it in an observable because I was initially thinking about implementing some of the "manage columns" functionality but that never got added so it doesn't have to be an observable here